discgolfbot/pdga/pdga_cog.py

```diff
@@ -0,0 +1,194 @@
+
+import nextcord
+from nextcord.ext import commands
+from nextcord.message import Attachment
+from nextcord import Interaction, utils, embeds, emoji, slash_command, SlashOption, SlashCommandOption, Member
+
+from discord_utils.embed_validation import *
+from typing import Optional
+from .pdgaPlayer import PdgaPlayer
+from .pdgaPlayerNumberRelations import *
+import re
+from scrapers.pdga import PlayerScraper, PdgaPlayerData
+
+class PdgaPlayerStat(commands.Cog): 
+    def __init__(self, discord_bot):
+        self.bot = discord_bot
+        self.config = {
+            "pdga_player_uri": "https://www.pdga.com/player/",
+            "db_file" : Path.joinpath(Path.cwd().parent, f'cfg/pdga_players.json') 
```

Comment:
Dette vil ikke åpne samme config mappe som ellers i applikasjonen. For å skrive til samme mappe må den skrive til "cfg/pdga_players.json". Dette vil da være en global config-fil.

Reply:
Hmm ja, jeg tror jeg ser hvorfor;
I `Dockerfile`settes workdir til /src (repo root) og så fyres boten til slutt med `python discgolfbot`, aka `__main__.py`, men workdir er repodir og ikke subdir(folder der __main__.py befinner seg) når boten fyres i docker. 

Med andre ord, når jeg tester debugging i vscode eller lignende så er mitt workdir repodir/discgolfbot, feil dir relativt til hvordan ting kjøres i prod. 
Fikser